client/app/components/faq/faqlist.js

```diff
@@ -59,26 +60,21 @@ class FaqList extends React.Component {
 			.catch(err => console.error(err));
 	}
 
-	renderListItem({ item, index }) {
+	renderListItem = ({ item, index }) => {
 		let rowBg = index % 2 == 1 ? FaqStyles.faqListItemOdd : null;
-
-		const icon = (
-			<Icon
-				name="chevron-thin-right"
-				size={30}
-				color="#787878"
-				style={{ marginTop: 5 }}
-			/>
-		);
-
 		return (
 			<ListItem
-				containerStyle={{ ...FaqStyles.faqListItem, rowBg }}
+				containerStyle={FaqStyles.faqListItem, rowBg}
 				titleStyle={FaqStyles.faqListItemText}
 				key={item.id}
 				title={item.question}
-				onPress={this.renderFaqDetails(item, index)}
-				rightIcon={icon}
+				onPress={() => this._renderFaqDetails(item, index)}
```

Comment:
Why the change?

Reply:
Overwriting due to merge conflicts again.